src/api/models/classification.py

```diff
@@ -0,0 +1,155 @@
+import uuid
+
+from django.conf import settings
+from django.contrib.gis.db import models
+from django.core.files.storage import FileSystemStorage
+from storages.backends.s3 import S3Storage
+
+from .base import BaseModel
+from .mermaid import BenthicAttribute, CollectRecord, GrowthForm, ObsBenthicPhotoQuadrat
+
+
+def select_image_storage():
+    if settings.DEBUG:
+        return FileSystemStorage()
+    else:
+        return S3Storage(bucket_name=settings.IMAGE_PROCESSING_BUCKET)
+
+
+class Label(BaseModel):
+    benthic_attribute = models.ForeignKey(
+        BenthicAttribute, related_name="labels", on_delete=models.CASCADE
+    )
+    growth_form = models.ForeignKey(GrowthForm, related_name="labels", on_delete=models.CASCADE)
+    description = models.TextField(null=True, blank=True)
+
+    class Meta:
+        unique_together = ("benthic_attribute", "growth_form")
+        db_table = "class_label"
+
+    def __str__(self):
+        return self.name
+
+    @property
+    def name(self):
+        return f"{self.benthic_attribute.name} {self.growth_form.name}"
+
+
+class Classifier(BaseModel):
+    name = models.CharField(max_length=50)
+    version = models.CharField(max_length=11)
+    patch_size = models.IntegerField(help_text="Number of pixels")
+    num_points = models.IntegerField(default=25)
+    description = models.TextField(max_length=1000, blank=True)
+    labels = models.ManyToManyField(Label, related_name="classifiers")
+
+    class Meta:
+        db_table = "class_classifier"
+
+
+class Image(BaseModel):
+    collect_record_id = models.UUIDField()
+    observation = models.ForeignKey(
+        ObsBenthicPhotoQuadrat, null=True, blank=True, on_delete=models.SET_NULL
+    )
+
+    image = models.ImageField(upload_to="", storage=select_image_storage)
```

Comment:
Let's chat about `upload_to` here and in thumbnail. Also, thinking ahead: the classification task will need to store feature vectors and hopefully an annotations csv with the image, so we'll need to ensure we can get the right path to the task so it stores in the right place.
![image](https://github.com/user-attachments/assets/eed171aa-afd2-4877-a5e1-f87cc639f630)

Reply:
Don't we still need to set this?